osu.Game/Screens/OnlinePlay/Multiplayer/Spectate/MultiSpectatorScreen.cs

```diff
@@ -178,19 +178,45 @@ protected override void Update()
         {
             base.Update();
 
-            if (!isCandidateAudioSource(currentAudioSource?.SpectatorPlayerClock))
-            {
-                currentAudioSource = instances.Where(i => isCandidateAudioSource(i.SpectatorPlayerClock)).MinBy(i => Math.Abs(i.SpectatorPlayerClock.CurrentTime - syncManager.CurrentMasterTime));
+            var maximizedPlayer = getMaximizedPlayer();
```

Comment:
please adjust spelling of `maximized` everywhere that it's used in this PR to `maximised` to be consistent with the project's spelling convention